.github/workflows/bridge-ui.yml

```diff
@@ -12,58 +12,35 @@ on:
 
 jobs:
   build-and-test:
-    runs-on: [taiko-runner]
-    steps:
-      - name: Checkout repository
-        uses: actions/checkout@v4
-      - name: Build and Checks
-        uses: ./.github/workflows/bridge-ui-ci.yml
+    uses: ./.github/workflows/bridge-ui-ci.yml
 
   # Deployment name follow the pattern: deploy_<appname(bridge-ui)>_<network(devnet|hekla|mainnet)>_<environment(preview|production)>
   deploy_bridge-ui_hekla_preview:
     if: ${{ github.ref_name != 'main' }}
-    runs-on: [taiko-runner]
```

Comment:
is this not needed or is it intentional? this is our heavier runner

Reply:
Yes, intentional. With reusable workflows we can just reference the target workflow without specifying a runner as we actually have no runnable steps in the job.